### ISSUE TITLE
Rendering Arabic ligatures without processing

### ISSUE THREAD

**Issue by minzhang2** [photopea/Typr.js#38]
I needed to render Arabic and found that the ligatures were not processed, so I introduced HarfBuzz and used it according to the instructions, but it didn’t work.
![image](https://user-images.githubusercontent.com/15793061/99635281-b66f7080-2a7c-11eb-8eb1-d892cd316383.png)
I want this
![image](https://user-images.githubusercontent.com/15793061/99635516-0e0ddc00-2a7d-11eb-804b-e1626a378b8a.png)

**Comment by photopea** [comment]
Hi, when you call shapeHB(), do you set the third parameter "ltr" to "false"?

**Comment by minzhang2** [comment]
No, the default value is "undefined", i do not think need to set "false"

**Comment by photopea** [comment]
Can you send me your font, please?

**Comment by minzhang2** [comment]
NotoSans-Bold.ttf
[font.zip](https://github.com/photopea/Typr.js/files/5565308/font.zip)

**Comment by photopea** [comment]
That is really strange. If you replace your font with DejaVu Sans (https://www.fontsquirrel.com/fonts/dejavu-sans) and keep your code without change, it is rendered correctly. I will have to look into it.

**Comment by minzhang2** [comment]
I tried, and it was amazing

**Comment by photopea** [comment]
@minzhang2 turns out your font has some old shaping mechanism. Could you use the latest font instead? Officially, you are supposed to use this font: https://github.com/googlefonts/noto-fonts/blob/master/hinted/ttf/NotoSansArabic/NotoSansArabic-Bold.ttf

**Comment by minzhang2** [comment]
I use fabricjs is ok，I don't understand why

**Comment by photopea** [comment]
In fabric JS, the text is rendered by a web browser (which supports old shaping mechanisms). Just replace your TTF file with the newer on.e

**Comment by minzhang2** [comment]
ok, very thanks
